account_payment_purchase/models/purchase_order.py

```diff
@@ -7,19 +7,31 @@
 
 class PurchaseOrder(models.Model):
     _inherit = "purchase.order"
+    _check_company_auto = True  # missing in the native purchase module !!!
```

Comment:
I think this side change shouldn't be done here.

Reply:
Of course ! It should be done in the purchase module !!!

Reply:
Yes, but not being in core, propose it there, but do another module for this, not put this on a traversal module that has nothing to do with it.

Reply:
done